docs/src/pages/components/data-grid/export/export.md

```diff
@@ -14,6 +14,16 @@ To enable the CSV export you need to compose a toolbar containing the `GridToolb
 
 {{"demo": "pages/components/data-grid/export/ExportSelectorGrid.js", "bg": "inline"}}
 
+### Customize which columns are exported
+
+By default, the CSV will only contain the visible columns of the grid.
+
+To include or hide other columns, there are two ways:
+
+1. Define the exact columns to be exported with the `fields` attribute in the `csvOptions` prop of [`GridToolbarExport`](/components/data-grid/components/#toolbar).
+   Set `allColumns` in `csvOptions` to true to include hidden columns, instead of only the visible ones.
```

Comment:
I think that we could make it even more obvious for the developers:

````suggestion
    ```jsx
   <GridToolbarExport csvOptions={{ fields: ['id', 'name'] }} />
   ```

   Set `allColumns` in `csvOptions` to true to include hidden columns, instead of only the visible ones.
````